packages/ui/src/pages/Home/MultisigView.tsx

```diff
@@ -1,50 +1,60 @@
 import AccountDisplay from '../../components/AccountDisplay'
 import { AccountBadge } from '../../types'
-import { renderMultisigHeading } from '../multisigHelpers'
 import MultisigActionMenu from './MultisigActionMenu'
 import { styled } from '@mui/material/styles'
-import { Chip, Paper } from '@mui/material'
+import { Box, Chip } from '@mui/material'
 import { useMultiProxy } from '../../contexts/MultiProxyContext'
+import Balance from '../../components/library/Balance'
+import { renderMultisigHeading } from '../multisigHelpers'
+import MultisigAccordion from './MultisigAccordion'
 
 const MultisigView = () => {
   const { selectedMultiProxy, selectedHasProxy, selectedIsWatched } = useMultiProxy()
 
   return (
-    <>
+    <MultisigViewWrapperStyled>
       <HeaderStyled>
         {selectedMultiProxy && (
-          <h3>{renderMultisigHeading(selectedMultiProxy.multisigs.length > 1)}</h3>
+          <h3>
+            {selectedHasProxy
+              ? 'Controlled by'
+              : renderMultisigHeading(selectedMultiProxy.multisigs.length > 1)}
+          </h3>
         )}
         <BoxStyled>{!selectedIsWatched && !selectedHasProxy && <MultisigActionMenu />}</BoxStyled>
```

Comment:
We **must** still see the 3dots buttons for watched accounts. This dropdown is currently used for renaming, and isn't visible for watched multisigs. This is the case why the `withSendButton` prop was created.